_posts/2016/09/2016-09-28-attendance-nz.md

```diff
@@ -11,49 +11,41 @@ Having taught and helped at a series of workshop over the past few months Tom Ke
 
 Please note that these are the authors' views and thus they should not be treated as representative for their home institutions.
 
-Tom based his opinions after having  taught at various workshops in Australia in New Zealand: <br/>
+Tom based his opinions after having taught at various workshops in Australia and in New Zealand, including Research Bazaar 2015 at the [University of Melbourne](http://unimelb.edu.au), Research Bazzar 2016 at the [University of Otago](http://www.otago.ac.nz/), University of Otago, [University of Canterbury](http://www.canterbury.ac.nz/), and [NeSI](https://www.nesi.org.nz/). There have been several other workshops in New Zealand facilitated by [NeSI](https://www.nesi.org.nz/), in Auckland, Wellington, and Palmerston North over the past year. 
 
-* Research Bazaar 2015 at the [University of Melbourne](http://unimelb.edu.au) and 2016 at the [University of Otago](http://www.otago.ac.nz/);
 
-* Several Software Carpentry workshops organised by the University of Otago, [University of Canterbury](http://www.canterbury.ac.nz/), and [NeSI](https://www.nesi.org.nz/), in February 2015, June 2015, and June 2016;
+### Main point
```

Comment:
GitHub Pages doesn't threat headers without space after `#` as headers. :-( Is good to keep the same style here as well.